cpp/src/IceGrid/SessionI.cpp

```diff
@@ -110,9 +112,9 @@ BaseSessionI::getGlacier2AdapterIdSet()
     return _servantManager->getGlacier2AdapterIdSet(shared_from_this());
 }
 
-SessionI::SessionI(const string& id, const shared_ptr<Database>& database, const IceInternal::TimerPtr& timer)
+SessionI::SessionI(const string& id, const shared_ptr<Database>& database, IceInternal::TimerPtr timer)
```

Comment:
I guess id was not updated by this auto-fix because it is own by the base class, I assume we will get other warnings that allow to easy fix the callers.